app/config/question-bank.js

```diff
@@ -726,15 +726,92 @@ const questionBank = {
           ],
           yarKey: 'plannedStorageCapacity'
         },
+        {
+          key: 'applying-for',
+          order: 110,
+          title: 'What are you applying for?',
+          pageTitle: '',
+          url: 'applying-for',
+          baseUrl: 'applying-for',
+          backUrlObject: {
```

Comment:
nextUrl should also be added for applying-for page, just for consistency. It also means we would only need one redirectUrl instead of 2 in the answers list

Reply:
I just added the next-url here and deleted one of the redirectUrl for consistency as you asked. it's ready for your review. Thank you.